src/github.com/getnelson/nelson/main.go

```diff
@@ -645,8 +647,8 @@ func main() {
 						cli.StringFlag{
 							Name:        "unit, u",
 							Value:       "",
-							Usage:       "Only list stacks that match a specified unit prefix",
-							Destination: &selectedUnitPrefix,
+							Usage:       "Only list stacks that match the specified unit",
+							Destination: &selectedUnit,
```

Comment:
I've added a new var for use when the given string is matched exactly against a unit, rather than a prefix. I've checked the API code to make sure it is an exact match, rather than a prefix.